src/entities/createPool/types.ts

```diff
@@ -0,0 +1,33 @@
+import { Address, Hex } from 'viem';
+
+export interface CreatePoolBase {
+    buildCall(input: CreatePoolInput): CreatePoolBuildCallOutput;
+}
+
+export type CreatePoolInput = WeightedCreatedPoolInput;
+
+export type WeightedCreatedPoolInput = {
```

Comment:
I see we have a few different naming standards on the code, for example `WeightedCreatedPoolInput` and `CreateWeightedPoolArgs`.

When refactoring Add/Remove Liquidity interface, I found it better to keep the same prefix everywhere, so for example:
- `WeightedJoinInput` -> `AddLiquidityWeightedInput`
- `NestedJoin` -> `AddLiquidityNested`

What do you think of following the same pattern of using `CreatePool...`?
- `CreatePoolBase`
- `CreatePoolInput`
- `CreatePoolWeightedInput`
- `CreatePoolWeightedArgs`

Reply:
This is still not following the pattern - it should be `CreatePoolWeigthedInput` and `CreatePoolWeightedArgs`.
This will make more sense when you start implementing Composable Stable pool create and have `CreatePoolComposableStableInput` and `CreatePoolComposableStableArgs`.
It's easier to follow when the prefixes are the same.